### PR TITLE
Haptic Feedback

### PR DESCRIPTION
**Pull request by KarlijndeJong**
I added some haptic driver code in the main.ccp. Can someone check my code :)
Still have to find out which pins to use for the connection between the haptic driver and the lolin.

### CODE REVIEW

**Comment by KarlijndeJong** [comment]
The vibration motors will play when the trigger is pressed. Right now it will vibrate all the time when the button is pressed, we only want it to vibrate when a gait is activated or stopped. I still have to change this.

**Comment by KarlijndeJong** [comment]
Code will let vibration motors vibrate when trigger is pressed.

**Comment by KarlijndeJong** [comment]
Pins for joystick have been changed.

File: platformio.ini [hunk 1]
```diff
@@ -14,5 +14,5 @@ board = lolin_d32_pro
 framework = arduino
 
 ; If Platformio gives a upload error, please specify this for your respective COM port
-upload_port = /dev/ttyUSB0
+upload_port = COM7
```
Comment (martijnvandermarel) [inline platformio.ini]: Change this back, is dependent on personal machnine, so not useful to push to develop.

**Review by martijnvandermarel — COMMENTED** [review]
Did you test this with (all) the prototype boards?

Also rename the examples/Vibration+button_example.cpp file because we removed the button part of this example code. Should be something like vibration_example.cpp or something.

File: platformio.ini (hunk 1 above)
Comment (KarlijndeJong) [reply platformio.ini]: Yes on all prototype boards the code works.

Comment (martijnvandermarel) [reply platformio.ini]: Great!

**Review by martijnvandermarel — APPROVED** [review]

**Pull request merged**